android/src/main/java/com/robinpowered/react/ScreenBrightness/ScreenBrightnessModule.java

```diff
@@ -33,14 +28,86 @@ public String getName() {
         return "ScreenBrightness";
     }
 
+    private boolean hasPermission() {
+        return Build.VERSION.SDK_INT < Build.VERSION_CODES.M ||
+                (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M &&
+                Settings.System.canWrite(getReactApplicationContext()));
+    }
+
+    private void requestPermission() {
+        Context context = getReactApplicationContext();
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && !Settings.System.canWrite(context)) {
+            Intent intent = new Intent(
+                    Settings.ACTION_MANAGE_WRITE_SETTINGS,
+                    Uri.parse("package:" + context.getPackageName())
+            );
+            mActivity.startActivity(intent);
+        }
+    }
+
+    private int getSystemBrightness() {
+        Integer brightness;
+        try {
+            brightness = Settings.System.getInt(
+                    getReactApplicationContext().getContentResolver(),
+                    Settings.System.SCREEN_BRIGHTNESS
+            );
+        } catch (Settings.SettingNotFoundException e) {
+            brightness = null;
+        }
+        return brightness;
+    }
+
+    private boolean setSystemBrightness(int brightness) {
```

Comment:
Could we throw a docblock onto this? I'm assuming we're returning whether the brightness was successfully changed or not?